core/site.go

```diff
@@ -412,8 +420,19 @@ func (site *Site) updateMeters() error {
 			soc, err := meter.(api.Battery).Soc()
 
 			if err == nil {
-				site.batterySoc += soc
-				site.log.DEBUG.Printf("battery %d soc: %.0f%%", i+1, soc)
+				var remainingCapacity float64
```

Comment:
welche Bedeutung hat remainingCapacity hier?

Reply:
Das ist die in der Batterie verbleibende Kapazität. Die wird für alle Betterien addiert und dann durch die Gesamtkapazität geteilt. Ergibt den aktuellen Ladezustand.

Reply:
Vielleicht besser

    remainingEnergy

ums klarer zu machen?

Reply:
Energie wird in Ws oder J angegeben. Hier haben wir es aber mit Ah zu tun. Das ist die Einheit der Kapazität oder Ladung.
`remainingEnergy` wäre also falsch. Wenn man es anders nennen möchte, dann müsste es `remainingCharge` heißen.
Ob das besser ist? Wäre ein neuer Begriff. 
Ich finde `remainingCapacity = soc*capacity ` eigentlich recht anschaulich.

Reply:
Wichtig ist doch lesbarer Code. Und an der Capacity der Batterie ändert sich nichts, sondern an der enthaltenen Energiemenge/Ladung. Da `charge` im Kontext schon anders besetzt ist gefiele mir `energy` besser.

> welche Bedeutung hat remainingCapacity hier?

Das war keine böse Absicht sondern es hat sich mir einfach aus dem Namen nicht erschlossen.

Reply:
Habe ich auch nicht so aufgefasst :-)

Aber Energy ist nicht gut, da definitiv falsch. 
Vielleicht reciht ja auch ein Kommentar für die nachfolgenden Generationen....